Package.swift

```diff
@@ -28,7 +28,7 @@ let package = Package(
     ],
     dependencies: [
         // Dependencies declare other packages that this package depends on.
-        .package(url: "https://github.com/apple/swift-nio.git", from: "1.0.0"),
+        .package(url: "https://github.com/apple/swift-nio.git", .branch("master")),
```

Comment:
[1.9.3](https://github.com/apple/swift-nio/releases/tag/1.9.3) is out :)

Reply:
@ianpartridge Thanks! I'm changing it now